webui/elm/src/View/SlotChart.elm

```diff
@@ -185,21 +186,22 @@ smallView =
     allocationChart smallConfig
 
 
+busySlots : List Types.Slot -> List Types.Slot
+busySlots =
+    List.filter (\slot -> slot.state /= Types.Free)
+
+
 allocationPercent : List Types.Slot -> String
 allocationPercent slots =
     let
         totalSlots =
             List.length slots
 
-        busySlots =
-            List.filter (\slot -> slot.state /= Types.Free) slots
-                |> List.length
-
         precision =
             10 ^ 1
 
         percentNum =
-            toFloat busySlots
+            (busySlots >> List.length >> toFloat) slots
```

Comment:
nice, I think this is where the unit test caught that 300% vs 33.3% discrepancy?

Reply:
yup was an issue with the order of arguments